fe-nc-news-react/src/components/Articles.jsx

```diff
@@ -1,22 +1,36 @@
 import { fetchArticles } from "../api";
 import { useState, useEffect } from "react";
 import ArticleCard from "./ArticleCard";
+import CircularProgress from "@mui/material/CircularProgress";
+import Box from "@mui/material/Box";
 
-export default function Articles() {
-    const [articles, setArticles] = useState([])
+export default function Articles({ isLoading, setIsLoading }) {
```

Comment:
try to have a loading state for each of the components that require the loading - if you plan to allow multiple requests to start or end the same loading state utilised by lots of components, some will start/stop loading at the wrong time! bring the loading state declaration down into this component.